api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
Re-commenting with what I put on the commit itself; shouldn't there be some notion of the current location as part of the state?  The location part should be read-only; should be most of the fields on `BreakPoint`